src/addScores.js

```diff
@@ -0,0 +1,17 @@
+class Score {
+  constructor(name, value) {
+    this.name = name;
+    this.value = value;
+  }
+}
+export default function addScores() {
+  const name = document.querySelector('.name-input').value;
+  const val = document.querySelector('.score-input').value;
+  if (name && val) {
+    const previousData = JSON.parse(localStorage.getItem('storedScores'));
+    previousData.push(new Score(name, val));
+    localStorage.setItem('storedScores', JSON.stringify(previousData));
+    // eslint-disable-next-line no-implied-eval
+    setTimeout('window.location.reload()', 0);
+  }
```

Comment:
- [ ] Let's use the [Leaderboard API ](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) and the `fetch` method coupled with `async` and `await` to send and receive data.